rest/changes_api.go

```diff
@@ -393,9 +393,12 @@ func (h *handler) sendSimpleChanges(channels base.Set, options db.ChangesOptions
 					if first {
 						first = false
 					} else {
-						h.response.Write([]byte(","))
+						_, _ = h.response.Write([]byte(","))
+					}
+					err = encoder.Encode(entry)
+					if err != nil {
+						base.Debugf(base.KeyChanges, "Error encoding change feed ebtry: %v", err)
```

Comment:
We're ignoring all the other encode errors, so maybe do the same here

Reply:
Fixed in latest revision.